### PR TITLE
Switch to better maintained spellcheck library

### PR DESCRIPTION
**Pull request by wesbiggs**
Problem
=======
yaspeller appears to be unmaintained, and was causing https://githubcom/LibertyDSNP/spec/security/dependabot/39.

Solution
========
spellchecker-cli is used instead; it works very similarly.

Change summary:
---------------
- [ ] Updated speller and regenerated dictionary and config files
- [ ] Fixed a handful of issues the new speller found that the old one did not

Steps to Verify:
----------------
1. Run `npm install`, then `npm run lint` or `npm run spellcheck`

### CODE REVIEW

**Review by wilwade — APPROVED** [review]
Looks great!

Tested it a few different ways and it works like a charm

File: .github/ISSUE_TEMPLATE/DSNP_Issue.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ Thanks for reporting an issue!
 ### Steps to reproduce
 Tell us how to reproduce this issue.
 
-### Expected behaviour
+### Expected behavior
```
Comment (shannonwells) [inline .github/ISSUE_TEMPLATE/DSNP_Issue.md:13]: 😆 I never caught this

**Review by shannonwells — APPROVED** [review]
omgtysm

File: .github/ISSUE_TEMPLATE/DSNP_Issue.md (hunk 1 above)
Comment (wilwade) [reply .github/ISSUE_TEMPLATE/DSNP_Issue.md:13]: British English accepted ;)

**Pull request merged**